### DISCUSSION TITLE
Github rendering only some pages in a site

### DISCUSSION THREAD

**Discussion by JJ** [community/community#23859] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve set up this <a href="https://github.com/Raku/Raku-Steering-Council" rel="noopener nofollow ugc">repo</a> so that the <code>main</code> branch is rendered. However, for some reason, <a href="https://raku.github.io/Raku-Steering-Council/papers/code_of_conduct" rel="noopener nofollow ugc">this specific page</a> is not. In that directory, there’'s one <a href="https://raku.github.io/Raku-Steering-Council/papers/Raku_Steering_Council_Code" rel="noopener nofollow ugc">page</a> that has been effectively rendered, while the others are not.</p>
<p>I’ve made some small cosmetic changes to force rendering, but that didn’t help. Meanwhile <a href="https://github.com/Raku/Raku-Steering-Council/runs/2115327215" rel="noopener nofollow ugc">builds</a> seem to be green, so I am really lost of what happens. Any idea?</p>

**Comment by ernest-phillips** [comment]
<p>Hi JJ,</p>
<p>I am looking into this. I do see that 3 of the 5 files from this <a href="https://github.com/Raku/Raku-Steering-Council/tree/main/papers">directory</a> are loading. Not sure why the others aren’t just yet.</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi JJ,</p>
<p>It looks like in this case it has to do with the capitalization of the file name. For a reason I haven’t discovered, files with caps names won’t get rendered.</p>

**Comment by JJ** [comment]
<p>Did you raise an issue with the corresponding GitHub unit? Is this still happening?</p>
